cmd/snap-seccomp/main.go

```diff
@@ -180,9 +180,7 @@ import (
 	"strings"
 	"syscall"
 
-	// FIXME: we want github.com/seccomp/libseccomp-golang but that
-	// will not work with trusty because libseccomp-golang checks
-	// for the seccomp version and errors if it find one < 2.2.0
+	// FIXME: we want github.com/seccomp/libseccomp-golang but that will not work with trusty because libseccomp-golang checks for the seccomp version and errors if it find one < 2.2.0
```

Comment:
I liked this better as it was before :-)

Reply:
as I tried to explain in the description gci  breaks the split one by keeping only the last line